modules/tide_content_collection/tide_content_collection.install

```diff
@@ -15,4 +18,65 @@ function tide_content_collection_install($is_syncing) {
   // Changing icons.
   $icon_filename = $icon_dir . DIRECTORY_SEPARATOR . 'content_collection.svg';
   _tide_set_paragraph_type_icon('content_collection', $icon_filename);
+
+  // Set content_collection permission to site_admin.
+  _assign_content_collection_perm_site_admin();
+}
+
+/**
+ * Updates to content_collection.
+ *
+ * Update content_collection paragraph label.
+ * Installs new content_collection_enhanced paragraph type.
+ * Assign access content_collection permission to site_admin.
+ */
+function tide_content_collection_update_8001() {
+  /** @var \Drupal\paragraphs\Entity\ParagraphsType $paragraph_type */
+  $paragraph_type = ParagraphsType::load('content_collection');
+  if ($paragraph_type) {
+    $paragraph_type->set('label', 'Content Collection (Raw JSON)');
+    $paragraph_type->save();
+  }
+
+  $configs = [
+    'paragraphs.paragraphs_type.content_collection_enhanced' => 'paragraphs_type',
+    'field.field.paragraph.content_collection_enhanced.field_content_collection_config' => 'field_config',
+    'core.entity_view_display.paragraph.content_collection_enhanced.default' => 'entity_view_display',
```

Comment:
@joshua-salsadigital the update failure above https://github.com/dpc-sdp/tide_api/pull/78/files#r876563005 may be caused by this.